core/storage/platform_parameter/gae_models_test.py

```diff
@@ -0,0 +1,83 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for core.storage.config.gae_models."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.platform import models
+from core.tests import test_utils
+import feconf
+
+(base_models, parameter_models) = models.Registry.import_models(
+    [models.NAMES.base_model, models.NAMES.platform_parameter])
+
+
+class PlatformParameterModelUnitTests(test_utils.GenericTestBase):
+    """Test PlatformParameterModel class."""
+
+    def test_get_deletion_policy(self):
```

Comment:
Ditto for these tests' names (e.g. ``test_deletion_policy_is_not_applicable``).

Reply:
I would probably keep it `test_get_deletion_policy` here since that is how it is used in the whole codebase, I think that for methods like these (which return a constant value) it is fine to have a simple named test.

Reply:
@vojtechjelinek We haven't been very disciplined about test names in the codebase historically, but this is something we're trying to improve going forward. Given this context, I'm not sure that "this is how it is used in the whole codebase" applies well to the case of test names -- I suggest figuring out what convention we should follow based on the merits of the various options for the test names themselves.

Reply:
Changed to 'test_deletion_policy_is_not_applicable'.

Reply:
This is actually something that's documented here: https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests. As Sean mentioned, we may have not consistently enforced this in the past, but we should move iteratively move the codebase toward our established conventions & best practices.